library/modules/bigip_gtm_wide_ip.py

```diff
@@ -409,6 +480,30 @@ def aliases(self):
         self._values['aliases'].sort()
         return self._values['aliases']
 
+    @property
+    def persistence(self):
+        if self._values['persistence'] is None:
+            return None
+        return str(self._values['persistence'])
+
+    @property
+    def persistence_ttl(self):
+        if self._values['persistence_ttl'] is None:
+            return None
```

Comment:
why is this here? only use property when you are actually checking or doing value transformation. Also if you have int values usually its best to check if the number is valid, check the device documentation for valid range, as far as implementation goes you need to look at other modules for established patterns use patterns

Reply:
added checking.